kong/error.go

```diff
@@ -1,18 +1,25 @@
 package kong
 
-type err404 struct {
+import (
+	"fmt"
+)
+
+type kongAPIError struct {
+	httpCode int
+	message  string
```

Comment:
This being unexported is good but we might be reaching a point where having an error type for this package is needed or having methods to retrieve further information from an error is needed.
This is simply a note for future development. Not changes required in this PR.